app/backend/wells/serializers_v2.py

```diff
@@ -286,9 +286,29 @@ class Meta:
             "static_water_level",
             "alternative_specs_submitted",
             "technical_report",
-            "drinking_water_protection_area_ind"
+            "drinking_water_protection_area_ind",
+            "licence_number"
         )
+    def get_licence_number(self, obj):
+        with connection.cursor() as cursor:
+            licence_num_query = """
```

Comment:
I'm looking for context on why we are using raw SQL here? I understand there are situations where routing around the model layer and using raw SQL is necessary.